idarling/core/events.py

```diff
@@ -130,7 +130,10 @@ def __init__(self, ea, new_name, old_name, local_name):
     def __call__(self):
         flags = ida_name.SN_LOCAL if self.local_name else 0
         if self.old_name:
-            self.ea = ida_struct.get_member_by_fullname(self.old_name).id
+            r = ida_struct.get_member_by_fullname(self.old_name) #if name like 'struct.member' the returning list. It's durty, need fix
+            if type(r) is list:
```

Comment:
Can you explain why you test the type to be a 'list' please?

Reply:
Some kind of mind eclipse i guess)).  Some major version ago this API worked differently. But, it doesn’t matter now. I saw the triggering of this event only in cases of renaming a global variable or function.  Guess, it doesn't work with some id for now. Only with offsets. Event and hook will be reworked.